<launch>
  <node pkg="turtlesim" type="turtlesim_node"  name="turtlesim_node"/>
  
  <node pkg="srvtry_pkg" type="turtle_move_client.py"  name="move_turtle_client"/>
  <node pkg="srvtry_pkg" type="turtle_move_server.py"  name="move_turtle_server"/>
  
</launch>
